packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -0,0 +1,45 @@
+import { GraphbackOperationType } from './GraphbackOperationType';
+
+// TODO Talk with Enda. Should mapping be dynamic and swapable or we going strong with CRUD spec
```

Comment:
So question. Are we going strong with CRUD model (I believe we should to keep things simple or allowing peole to override it if needed and provide custom names. This is very simple change in terms of code but it has tons of side impacts on docs etc.